distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Column_Naming_Helper.enso

```diff
@@ -104,7 +104,8 @@ type Column_Naming_Helper
     to_expression_text self value =
         if is_column value then "[" + value.name + "]" else
             if value.is_a Regex then "r/" + (value.pattern.replace "/" "\/") + "/" else
-                value.pretty
+                if Meta.is_atom value then ".."+value.to_text else
+                    value.pretty
```

Comment:
I'm afraid that this won't work correctly if the atom has any fields.

For example:
```
> (Case_Sensitivity.Insensitive Locale.uk).to_text
>>> (Insensitive en-GB)
```
So `to_text` plus `..` will yield here `..(Insensitive en-GB)` which is not a valid expression. I imagine it should be `(..Insensitive 'en-GB')` if we want it to be correct (although this example will still not work as we don't have a way to convert a text to Locale (although probably should be easy to create a `Locale.from (that:Text)` conversion)).

Reply:
Worth noting that doesn't work in expression and the current `pretty` is bust.

```
> Case_Sensitivity.Insensitive Locale.uk . pretty
>>> (Case_Sensitivity.Insensitive en-GB)
```

Reply:
Fixed with James suggestion. If anyone knows where we are using this then I will add a test.